evap/evaluation/tests/test_views.py

```diff
@@ -183,3 +183,35 @@ def test_edit_display_name(self):
 
         page = self.app.get(self.url, user=self.responsible)
         self.assertContains(page, "testdisplayname")
+
+
+class TestNegativeLikertQuestions(WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        cls.voting_user = baker.make(UserProfile, email="voting_user1@institution.example.com")
+        cls.contributor = baker.make(UserProfile, email="contributor1@institution.example.com")
```

Comment:
Do you really need a contributor? Shouldn't a question in the general questionnaire be just fine?